notebooks/03automate.py

```diff
@@ -0,0 +1,63 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.1
+#   kernelspec:
+#     display_name: 'Python 3.7.10 64-bit (''workflow-miniscope'': conda)'
+#     metadata:
+#       interpreter:
+#         hash: fb69438121b2de1073146525f02c3c959a56b9d2c1d6b0a62c34f01201323d1c
+#     name: python3
+# ---
+
+# # [WIP] Automated workflow
```

Comment:
In the next round, let's add some descriptions on what this notebook does and the context when to use this notebook.